app/views/brexit_checker/_stay_updated.html.erb

```diff
@@ -1,4 +1,6 @@
-<h2 class="govuk-heading-l stay-updated__heading"><%= t("brexit_checker.stay_updated.heading") %></h2>
+<h2 class="govuk-heading-m stay-updated__heading">
```

Comment:
I'm not sure the `stay-updated__heading` class is being used?

Reply:
Ah, I see it's removed in a later commit. I would suggest squashing the following commits into one "Update subscribe section content" kind of commit, as I'm not sure it really needs to be broken down so much - we wouldn't really want to deploy one commit without the others:

- `Change subscription heading to medium heading`
- `Change subscription description to simple p tag`
- `Update new post-election copy for subscription section`
- `Change stay updated description css`

Reply:
Thanks very much, have squashed those